sdk/keyvault/keyvault-certificates/test/internal/userAgent.spec.ts

```diff
@@ -2,18 +2,25 @@
 // Licensed under the MIT license.
 
 import * as assert from "assert";
-import { version } from "../../package.json";
 import { SDK_VERSION } from "../../src/generated/utils/constants";
 import { packageVersion } from "../../src/generated/keyVaultClientContext";
+import { isNode } from "@azure/core-http";
+import path from "path";
+import fs from "fs";
 
-describe("Certificates client's user agent", () => {
-  // The tests follow
-
-  it("the version at constants should be as expected", async function() {
-    assert.equal(version, SDK_VERSION);
+describe("Certificates client's user agent (only in Node, because of fs)", () => {
+  it("SDK_VERSION and packageVersion should match", async function() {
+    assert.equal(SDK_VERSION, packageVersion);
```

Comment:
this seems like a great thing for devtool to eventually provide as a lint check @willmtemple